task.sql

```diff
@@ -1,11 +1,14 @@
--- Use our database
 USE ShopDB; 
 
--- Some data should be created outside the transaction (here)
+INSERT INTO Orders (CustomerID, Date) 
+VALUES (1, '2023-01-01');
```

Comment:
The `INSERT INTO Orders` statement is executed outside of the transaction. Consider moving this statement inside the transaction block to ensure that all operations are atomic and can be rolled back if something goes wrong.